mlflow/lightgbm.py

```diff
@@ -284,12 +321,18 @@ def callback(env):
         for imp_type in ['split', 'gain']:
             features = model.feature_name()
             importance = model.feature_importance(importance_type=imp_type)
+            try:
+                log_feature_importance_plot(features, importance, imp_type)
+            except Exception:  # pylint: disable=broad-except
+                _logger.exception('Failed to log feature importance plot. LightGBM autologging'
```

Comment:
```suggestion
                _logger.exception('Failed to log feature importance plot. LightGBM autologging '
```

Micronit: just making sure there are spaces between words in the error message

Reply:
Oh, thanks for the catch!

Reply:
Fixed it.